api/v1/users/routes.js

```diff
@@ -17,6 +17,19 @@ module.exports = [
     }
   },
   {
+    method: 'PUT',
```

Comment:
Lets remove PUT. We are not going to use this anytime soon and moreover this is referencing `get.js` , which causes confusion to the reader.